<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPVanDialog</class>
 <widget class="QDialog" name="addPVanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add private van</string>
  </property>
  <widget class="QDialogButtonBox" name="addPVanButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_PV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="licnr_PV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Owner Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owname_PV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Owner Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ownsur_PV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Owner Civic ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ownid_PV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>Load capacity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="locap_PV">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addPVanButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addPVanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPVanButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addPVanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
